src/include/clSPARSE.h

```diff
@@ -360,6 +360,40 @@ clsparseDCsrMatrixfromFile( clsparseCsrMatrix* csrMatx, const char* filePath, cl
         const cldenseVector* y,
         const clsparseControl control );
 
+    // functions for tests of scan transformation, as simple as possible
```

Comment:
You are right, I should delete these interface functions, as written in the comment lines they are just for testing purposes. I exposed them to easily test the algorithms with the gtest. I did not found better solution for testing internal code.